code/modules/research/designs/boards/machine_engie.dm

```diff
@@ -14,6 +14,16 @@
 	category = "Machine Boards"
 	build_path = /obj/item/weapon/circuitboard/smes
 
+/datum/design/treadmill
```

Comment:
It's a generator. If you disagree with it, PR a removal of RUST and PACMAN from this file.